.buildkite/pipeline.yml

```diff
@@ -0,0 +1,39 @@
+env:
+  JULIA_VERSION: "1.7.2"
+  OPENMPI_VERSION: "4.0.4"
+  CUDA_VERSION: "11.2"
+  OPENBLAS_NUM_THREADS: 1
+  BUILDKITE_COMMIT: "${BUILDKITE_COMMIT}"
+  BUILDKITE_BRANCH: "${BUILDKITE_BRANCH}"
+  JULIA_DEPOT_PATH: "${BUILDKITE_BUILD_PATH}/${BUILDKITE_PIPELINE_SLUG}/depot/cpu"
```

Comment:
```suggestion
```

Let's skip out on using the Julia depot until we get its consequences documented somewhere.